backend/src/apiserver/client_manager.go

```diff
@@ -367,6 +378,19 @@ func createMinioBucket(minioClient *minio.Client, bucketName, region string) {
 	glog.Infof("Successfully created bucket %s\n", bucketName)
 }
 
+func initLogArchive() (logArchive archive.LogArchiveInterface) {
+	logFileName := common.GetStringConfigWithDefault(
+		"ArchiveConfig.LogFileName", os.Getenv(archiveLogFileName))
```

Comment:
common.GetStringConfigWithDefault is already getting from env var. You can read viper's documentation.

For `ArchiveConfig.LogFileName`, corresponding env var name will be `ARCHIVECONFIG_LOGFILENAME`, so I'd suggest looking at my other comment to make it env var friendly.

Reply:
Done, thanks for comment.